### PR TITLE
readme update: it's version 1.0 already

### PR DESCRIPTION
**Pull request by e-moe**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Homebrew installs the stuff you need that Apple didn’t.
 - [Gems, Eggs and Perl Modules](Gems,-Eggs-and-Perl-Modules.md)
 - [Homebrew and Python](Homebrew-and-Python.md)
 
-Please note: Homebrew is not 1.0 yet. Generally it works well, but when
+Please note: Homebrew generally works well, but when
 it doesn’t we’d hope you report the bug.
```
Comment (zmwangx) [inline docs/README.md]: It would be nice if you could also add a new line after the paragraph, thanks. And refill the lines to 70 columns maybe.

Comment (e-moe) [inline docs/README.md]: @zmwangx done

Comment (e-moe) [inline docs/README.md]: do I need to squash commits?

Comment (zmwangx) [inline docs/README.md]: Yes.

Comment (e-moe) [inline docs/README.md]: squashed

Comment (ddebernardy) [inline docs/README.md]: Might be just me, but "Please note: Homebrew generally works well, _but_..." reads like "Heads up, this software will blow up in your face." The intent of the original message was to say "heads up, you're using beta software and we'd love to hear the bugs you find." IMHO drop the whole paragraph altogether. Anyway, my $.02.

Comment (zmwangx) [inline docs/README.md]: The paragraph says if you see bugs please report them. Stable software has bugs too.

**Comment by MikeMcQuaid** [comment]
Nice spot, thanks!

**Pull request merged**